platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/NativeMapView.java

```diff
@@ -61,11 +61,8 @@
   // Listener invoked to return a bitmap of the map
   private MapboxMap.SnapshotReadyCallback snapshotReadyCallback;
 
-  //
-  // Static methods
-  //
-
-  static {
+  @Override
+  public void loadNativeLibrary() {
```

Comment:
I would prefer to use Interfaces in any case for testability. Instead of depending on classes directly.

For the library loading, I would like to centralise it, regardless of testability. For example, create a class `LibraryLoader`, with a single static method `load()` which can be called from the classes containing methods from a static initialisation block still. This would centralise the knowledge about the library name at least. And also, you could mock this method with PowerMock.